docs/literate.jl

```diff
@@ -40,7 +53,37 @@ function preprocess(content)
     # remove VSCode `##` block delimiter lines
     content = replace(content, r"^##$."ms => "")
 
-    return content
+    # adds the current version of the packages
+    append = """
+    # ### Package and system information
+    # #### Package version
+    # ```julia
+    $(chomp(replace(pkg_status, r"^"m => "# ")))
+    # ```
+    # #### Computer information
+    # ```julia
+    $(chomp(replace(sprint(InteractiveUtils.versioninfo), r"^"m => "# ")))
+    # ```
+    # #### Manifest
+    # Click [here to download the Manifest.toml]($(MANIFEST_OUT)) for this notebook's package environment
```

Comment:
It seems the link is broken in the Jupyter notebook version, e.g., https://nbviewer.org/github/JuliaGaussianProcesses/ApproximateGPs.jl/blob/gh-pages/previews/PR97/examples/b-classification.ipynb. I think ideally this should be hardcoded to the documentation webpage since otherwise the link will break if one only copies the webpage or Jupyter notebook without the Manifest.toml.

I think this would need https://github.com/fredrikekre/Literate.jl/issues/151 to work properly for different branches and versions.

Reply:
Accessing the Manifest.toml from the link you posted works just fine for me!

Reply:
It works fine for me too!

Reply:
@devmotion Could you check again? I think the whole nbviewer setup can be a bit slow sometimes (for some reason)

Reply:
Now it works for me as well :slightly_smiling_face: Sorry for the noise, I guess nbviewer might have used an outdated cache: https://nbviewer.org/faq#why-is-nbviewer-showing-an-outdated-version-of-my-notebook